bastion.yml

```diff
@@ -111,5 +111,11 @@
               document_type: apache
               paths:
                 - /var/log/apache/*.log
+        - name: ansible_runner
+          prospectors:
+            - input_type: log
+              document_type: ansible_runner
+              paths:
+                - "/var/www/html/cron-logs/$env/*.log"
```

Comment:
We should only track either the current or the latest log, but if we track all logs in the directory then we will get a bunch of duplicate entries after every log rotation. Also there is nothing setting `$env`, so that should be another `*` instead.

Also, the document type is used by logstash to know how to parse the logs. Using `ansible_runner` for the type is correct, but doesn't have a matching logstash filter yet, we will need to add that as well. We will need a change similar to https://github.com/BonnyCI/hoist/commit/808c70cd50981dc9f5eaef560aacf9af02faacef to add the new log parsing. And we can probably reuse the exact same multiline configuration from that change since it is based on lines starting with a timestamp. The tricky part is the grok pattern, we can chat about that in irc or slack.